### ISSUE TITLE
Magic bow registering as normal bow

### ISSUE THREAD

**Issue by BaanDar** [elBukkit/MagicPlugin#723]
The spells for the magic bow dont work like they should- the game detects it as a regular bow. Shooting it will just shoot a regular arrow, even if a spell arrow is selected, and if the spell inventory is open on the hotbar it will consume the arrow used as an icon for the arrow_regular spell. Switching the id to something else fixes that issue, but the spells still don't cast.
Changing the magic bow to work as a quick cast and changing the id for arrow_regular to something else allows the spells to work, but they do not shoot based of a bow draw value. Is there any way to fix this? Not sure if anyone else has this problem either.

**Comment by NathanWolf** [comment]
Are you pulling the bow all the way back like you would to fire an arrow normally?

**Comment by BaanDar** [comment]
Yes, i am

**Comment by NathanWolf** [comment]
Seems ok on my dev server- have you made any modifications to your configs for the bow or spells?

I just did
```
/mgive magicbow
/mgive arrow 64
Hit Q
Select the poison arrow
bull bow back, fire poison arrow
```

**Comment by BaanDar** [comment]
Tried that, didn't work- same type thing happening. I hadn't modified configs relating to the bow or arrow spells at all, but I reverted to all default configs by regenerating the config files and still had the same problem. Tested it on a friend's server and it worked for him too. I'll attach a video of the bow in-game and a screenshot of my plugins list. Starting to wonder if it'd be a plugin incompatibility, but not sure what plugins would interfere.

![Plugins list](https://user-images.githubusercontent.com/68800057/88499294-659ab000-cf93-11ea-951a-45a434cef290.jpg)

https://www.dropbox.com/s/tpngimklzk1xgy2/1.16.1%20-%20Magic%20bow.mp4?dl=0
